.gitignore

```diff
@@ -1,2 +1,77 @@
-node_modules/
-npm-debug.log
+### Node template
```

Comment:
i just use intellij, they have a plugin that generates from templates, this is the JetBrains and Node template.